README.md

```diff
@@ -0,0 +1,129 @@
+# StackStorm in Docker containers
+
+The initial container configuration is as follows:
+
+ - st2-upstart (st2 + st2web + sshd)
+ - mongo
+ - rabbitmq
+
+## Usage
+
+Build the st2-upstart image:
+
+  ```
+  make build
+  ```
+
+Start the docker environment:
+
+  ```
+  docker-compose up
+  ```
+
+Install the SSH keys locally, and setup ST2 user (this only needs to be run once per `docker-compose up`):
+
+*NOTE: The default values for `ST2USER` and `ST2PASSWORD` are `st2admin` and `Ch@ngeMe`
+respectively. You only need to specify these variables if you do not want to use the default
+values.*
+
+  ```
+  make setup ST2USER=st2admin ST2PASSWORD=Ch@ngeMe
+  ```
+
+Use either `ssh` or `docker exec` to connect to the st2-upstart container:
+
+  ```
+  ssh -i ~/.ssh/id_busybee root@localhost
+  ```
+
+  ```
+  docker exec -it st2docker_st2-upstart_1 /bin/bash
+  ```
+
+Once connected to the st2-upstart container, verify that `st2-self-check` passes successfully.
+
+*NOTE: For now, the mistral tests will fail because st2mistral is not yet configured.*
+
+  ```
+  st2ctl reload
+  . ~/st2.vars
+  st2-self-check
+  ```
+
+*NOTE: The `./packs` directory is mounted into the `st2-upstart` container at `/opt/stackstorm/packs`.*
```

Comment:
This is messy. I understand what you've done
The problem is, without system packs you can't do anything. E.g. packs pack is needed to set up virtual_env. I lost them first thing as I begin to play. Surely recovered it (copy from .pkg, run st2ctl reload) but it's not smooth.

As a user I want to add my packs. To some existing standard ones.
I DO like the packs installed with `pack install` to show up on my hosts' disk (as I will configure it, and modify).
I just don't like the step where I need to copy packs over from packs.pkg to packs.


How about we use [the option to provide additional path for loading packs](https://github.com/StackStorm/st2/blob/master/conf/st2.conf.sample#L83), and `VOLUME /opt/stackstorm/packs.dev` or something in Dockerfile to specify user packs, and keep the defaiult/system packs intact? Or even the other way around - use `system_packs_base_path=/opt/stackstorm/packs.pkg` and `packs_base_paths=/opt/stackstorm/packs`?

Note that the secondary path for packs is not commonly used and may have regressed into no longer working.